media/js/firefox/new/experiment-search-retention-funnelcakes.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var experimentId = 'experiment_search_retention_funnelcake';
+
+    // swiped from mozilla-client.js
+    var ua = navigator.userAgent;
+    var isLikeFirefox = /Iceweasel|IceCat|SeaMonkey|Camino|like\ Firefox/i.test(ua);
+    var isFirefox = /\s(Firefox|FxiOS)/.test(ua) && !isLikeFirefox(ua);
+    var isMobile = /^(android|ios|fxos)$/.test(window.site.platform);
+
+    var hasCookie = Mozilla.Cookies.hasItem(experimentId);
+    var fcRe = /^.*\?.*f=\d{3}.*/;
+
+    // 0. if visitor already has a cookie for this experiment, skip the expenisve checks below
+    if (hasCookie) {
+        runExperiment();
+    }
+    // 1. check to make sure user is on windows and not fx
+    // 1.1 extra check around DNT so we don't do unnecessary geolookups
+    // 1.2 make sure user isn't on a variation
+    else if (!fcRe.test(document.location.search) && window.site.platform === 'windows' && !isFirefox && !isMobile && typeof Mozilla.dntEnabled === 'function' && !Mozilla.dntEnabled()) {
+        // 2. check geolocation for US
+        var xhr = new XMLHttpRequest();
+
+        xhr.onload = function(r) {
+            if (r.target.status >= 200 && r.target.status < 300) {
+                var country;
+
+                try {
+                    country = r.target.response.country_code.toLowerCase();
+                } catch (e) {
+                    // unset var above is good enough
+                }
+
+                if (country === 'us') {
+                    runExperiment();
+                }
+            }
+        };
+
+        xhr.open('GET', 'https://location.services.mozilla.com/v1/country?key=a9b98c12-d9d5-4015-a2db-63536c26dc14');
```

Comment:
This throws an error in older IE browsers due to same origin policy, do we need to exclude those?

<img width="577" alt="screen shot 2017-06-20 at 10 36 10 am" src="https://user-images.githubusercontent.com/400117/27326810-72c80078-55a4-11e7-8a27-5e0f5df036f4.png">